bp_common/syn/Makefile.vcs

```diff
@@ -128,6 +128,16 @@ regress_beebs.v: $(BEEBS_REGRESSION_V)
 $(BEEBS_REGRESSION_V):
 	$(MAKE) sim.v PROG=$(basename $@)
 
+LCFG_2=e_bp_dual_core_cfg
+LCFG_4=e_bp_quad_core_cfg
+LCFG_8=e_bp_oct_core_cfg
+LCFG_16=e_bp_sexta_core_cfg
+
+regress_mc.v: $(MC_REGRESSION_V)
+$(MC_REGRESSION_V):
+	$(eval CFG_N := $(shell echo $(basename $@) | sed -r "s/(^.*_)//g"))
```

Comment:
What's this doing exactly?

Reply:
It extracts the number from the file name. For example, if the file is demo_2.v, it sets CFG_N = 2. Then this number gets combined with the text "LCFG_" to set the CFG parameter passed to "make sim.tool CFG=xxx"

Like I said, it is a bit of a hack. But we need to set the CFG variable for the simulation properly since these tests don't use the default CFG (single core) defined in the bp_top/syn Makefile

Reply:
Ohh, I get it now.  Could we override it in the gitlab yaml instead?

Reply:
I made an adjustment. But does the new way I have it written allow the multicore tests to run concurrently?